api/migrations/versions/794506c08e10_.py

```diff
@@ -0,0 +1,53 @@
+"""empty message
+
+Revision ID: 794506c08e10
+Revises: 1afd8dfc8d05
+Create Date: 2021-08-16 11:37:15.501414
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '794506c08e10'
+down_revision = '1afd8dfc8d05'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('review_score', sa.Column('is_active', sa.Boolean, nullable=True))
+    op.add_column('guest_registration_answer', sa.Column('is_active', sa.Boolean(), nullable=True))
+    op.add_column('answer', sa.Column('is_active', sa.Boolean(), nullable=True))
+
+    op.add_column('review_score', sa.Column('updated_on', sa.TIMESTAMP, nullable=True,
+                                            server_default=sa.func.now(), onupdate=sa.func.now()))
+    op.add_column('guest_registration_answer', sa.Column('updated_on', sa.TIMESTAMP, nullable=True,
+                                                         server_default=sa.func.now(), onupdate=sa.func.now()))
+    op.add_column('answer', sa.Column('updated_on', sa.TIMESTAMP, nullable=True,
+                                      server_default=sa.func.now(), onupdate=sa.func.now()))
```

Comment:
None of these columns have been added to the models. Was this intentional? I can see they would default to the current date and the columns and values would exist in the database but I'm uncomfortable with them being _hidden_ in these scripts especially since we're using a model/code first approach. What do you think about moving these to the model class and also explicitly defining them there?

Reply:
This definitely makes sense @amritpurshotam . I overlooked defining them in the model class because I saw they are not used in any part of the code because the date is only set and not modified anywhere. Let me declare them in the models too.

Reply:
Sorry missed this update 😓 I see they're on the model class now, thanks!
There is one more thing though, I should have mentioned this on the last comment so apologies for the inconvenience.
Can we please rename the column to `created_on` instead and remove the `onupdate=sa.func.now()` bit (in fact I'm in two minds if we should even have the `server_default` and just explicitly set the field manually since that's what's done elsewhere in the codebase)? The reason is that we don't want this value to ever change (especially accidentally) and having the name reflect that would help communicate this rule. 

P.S. you mentioned that the field isn't used anywhere which is a fair assumption. But it will become important later for when we perhaps want to display all the answers (for say an admin of some kind), it would be nice to order them by the creation date and having the field on the model will be required for this. A second reason is that any database changes we make must flow from the codebase (and never the other way round). We have documented the process [here](https://github.com/deep-learning-indaba/Baobab/tree/develop/api#database-migrations) with the commands to run to generate the initial migration. We then edit it if necessary, for eg. with new not null columns, or we edit the code again to make sure the migration we want gets generated correctly). We shouldn't normally be writing migrations from scratch.

Reply:
Updated with the new recommendations. `server_default` left for now.

Reply:
@bdhobare Thanks for the update! Approved from my side 😄